### PR TITLE
Adding notes on communication differences to review culture document

### PR DESCRIPTION
**Pull request by vsoch**
This is hard to articulate, but I think important to add (it's hit me several times in various open source communities, and it helps to have awareness for it). Basically:

 - people vary in how they communicate.
 - if expectations are not lined up (for example, I expect criticism wrapped in complements and an interaction is more straight forward) this can lead to hurt feelings or conflict
 - we have to encourage folks to be aware of the differences, so if a conflict arises, they can either realize "It's not me, this person just communicates more straight forward" or "This isn't how I want these interactions to go, I want to open a dialogue about it."

it's a hard thing to write up, because there isn't a single right way or suggestion to make for how to best communicate. But I do think entering most interactions with this awareness is helpful and important.

## Organization

As for organization, I changed the last section (which was originally just Best Practices for Reviewers) to be about Communication, and then the best practices is a subsection (because it does seem to be all about communication). At some future point someone could add a "Best Practices for Contributors" as well. Another organizational idea would be to have a general "Best Practices" section, separate from the "Communication" section, something like this:

- Communication
  - Be aware of culture
- Best Practices
  - Best Practices for Reviewers
  - Best Practices for Maintainers
  
The above would allow best practices to be included that are not directly communication related.
Signed-off-by: Vanessa Sochat <vsochat@stanford.edu>

### CODE REVIEW

File: culture/review_culture.md [hunk 1]
```diff
@@ -147,3 +172,26 @@ doing this, you should ask: in what ways is doing it my way materially better?
...
+and isn't quite sure how to best help. In both cases, as a maintainer you should
+take every measure to engage the contributors. For example, although this would
+be a respectful response:
+
+> Thank you for your contribution! We've decided to go in another direction, so unfortunately we will not be integrating the change.
+
+It ends the participation right away. Instead, you might consider
+a statement that delivers the same message, but further engages the contributor:
+
+> Thank you for your contribution! We've decided to go in another direction, and although we cannot use the changes here, we are interested in your thoughts on the idea here. Would you like to help?
```
Comment (BrianLitwin) [inline culture/review_culture.md]: I have a slight preference for the example you gave in your original comment, "Would you be interested in working on [well-scoped task]?", as a better template compared to "we are interested in your thoughts on the idea here". Not requesting a change, just a thought.

File: culture/review_culture.md [hunk 2]
```diff
@@ -120,16 +120,41 @@ Some changes don't ever merge. A few good reasons why this may happen include:
...
+communication. For example, in Western culture it's common to wrap criticism
+with a complement: 
+
+> "You did a really good job on X, _but_ I wonder if we could talk about your decision for using this data structure?" 
+
+Other cultures might be more straight forward to directly give more blunt feedback:
+
+> "The algorithm could be improved by this amount by using this data structure."
+ 
+If you are expecting differently, it can feel mean when it is not intended to.
```
[the remark this answers is not in the document]
Comment (vsoch) [reply culture/review_culture.md]: Changing to `the critique can feel mean`

File: culture/review_culture.md [hunk 3]
```diff
@@ -120,16 +120,41 @@ Some changes don't ever merge. A few good reasons why this may happen include:
...
+with a complement: 
+
+> "You did a really good job on X, _but_ I wonder if we could talk about your decision for using this data structure?" 
+
+Other cultures might be more straight forward to directly give more blunt feedback:
+
+> "The algorithm could be improved by this amount by using this data structure."
+ 
+If you are expecting differently, it can feel mean when it is not intended to. 
+Thus, while the community encourages open, and respectful communication,
```
[the remark this answers is not in the document]
Comment (vsoch) [reply culture/review_culture.md]: Removing comma.

File: culture/review_culture.md [hunk 4]
```diff
@@ -120,16 +120,41 @@ Some changes don't ever merge. A few good reasons why this may happen include:
...
+
+Other cultures might be more straight forward to directly give more blunt feedback:
+
+> "The algorithm could be improved by this amount by using this data structure."
+ 
+If you are expecting differently, it can feel mean when it is not intended to. 
+Thus, while the community encourages open, and respectful communication, 
+it's good to keep in mind that this definition might vary depending on an individual's
+background and previous experience. Regardless, disrespect or communication that is
+hurtful is not ok, and if you feel offended by an interaction,
```
[the remark this answers is not in the document]
Comment (vsoch) [reply culture/review_culture.md]: I like how this reads better too, changed locally.

File: culture/review_culture.md (hunk 1 above)
Comment (vsoch) [reply culture/review_culture.md]: Fixed!

File: culture/review_culture.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (vsoch) [reply culture/review_culture.md]: And now pushed!

**Comment by vsoch** [comment]
> In the spirit of the "prefer small changes" section to the guide (which we haven't added yet stuck_out_tongue) -- would you be open to making PRs like this into two PRs in the future? 

Yes most definitely - I realized this after I pushed and grimaced a little - I will keep them separate in the future! Looking at your other comments now (and will address and update the PR).

File: culture/review_culture.md [hunk 5]
```diff
@@ -120,16 +120,41 @@ Some changes don't ever merge. A few good reasons why this may happen include:
 - Over the course of review, the contributor or reviewers may find a different
   and better way to achieve the same goals
 
-# Best Practices for Reviewers
+# Communication
```
[the remark this answers is not in the document]
Comment (vsoch) [reply culture/review_culture.md]: okay for sections that we haven't written I'll add them (empty) and save the content for a future PR. Not making that mistake again :)

File: culture/review_culture.md [hunk 6]
```diff
@@ -147,3 +172,26 @@ doing this, you should ask: in what ways is doing it my way materially better?
 What benefits come from the changed approach? If you don't have clear answers
 to these questions, don't ask for the change.
 
+### Take every measure to include contributors
```
[the remark this answers is not in the document]
Comment (vsoch) [reply culture/review_culture.md]: I think I've scoped it to be the more specific case, "Include contributors even when closing their changes" so I'll update to that for now - if it gets more generalized with further examples, then we can change back.

**Pull request merged**